src/integrationTest/java/org/opensearch/security/Song.java

```diff
@@ -0,0 +1,44 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*
+*/
+package org.opensearch.security;
```

Comment:
Can we move this to a test package?

Reply:
Actually, I prefer to keep this class close to tests which use data stored in the class. Thus the class can be packaged private. What do you think?